YIF_XUnitTests/Integration/YIF_Backend/Controllers/UsersControllerTests.cs

```diff
@@ -102,15 +103,17 @@ public async Task Send_ResetPassword_IfEmail_InCorrect(string email)
             string confirmPassword,
             string recaptcha)
         {
-            // Act
+            //Arrange
+            _userInputAttribute.SetUserIdForHttpContext();
```

Comment:
you should also rename these tests, cause there is a problem with "Current", I guess it must be replaced to "Correct"